_includes/usagepolicy.html

```diff
@@ -0,0 +1,32 @@
+<div class="full-width-bg component" xmlns="http://www.w3.org/1999/html">
+  <div class="grid-wrapper">
+    <div class="width-6-12 width-12-12-m">
+      <p>This page contains the policy for the Red Hat sponsoered anonymous data collection. See <a href="/usage">Usage</a> page for details.</a></p>
+      <p class="intropara">In connection with your use of this Red Hat offering, Red Hat may collect telemetry data about your use of the software. This data allows Red Hat and community extension writers to understand how the software is being used and to improve Red Hat offerings and support, including identifying, troubleshooting, and responding to issues that impact users. The data may be shared internally within Red Hat to improve the user experience.</p>
```

Comment:
This paragraph is very worrying. I tried to tone it down a bit. I know we describe precisely what we collect later but you can't be sure people will read it all.
```suggestion
      <p class="intropara">In connection with your use of this Red Hat offering, Red Hat may collect build-time telemetry data about your use of the software. This data allows Red Hat and community extension writers to understand how the software is being consumed and to improve Red Hat offerings and support, including identifying, troubleshooting, and responding to issues that impact users. The data may be shared internally within Red Hat to improve the user experience.</p>
```